packages/contracts-bedrock/contracts/universal/OptimismMintableERC20.sol

```diff
@@ -12,7 +12,7 @@ import "./SupportedInterfaces.sol";
  *         Designed to be backwards compatible with the older StandardL2ERC20 token which was only
  *         meant for use on L2.
  */
-contract OptimismMintableERC20 is ERC20 {
+contract OptimismMintableERC20 is IOptimismMintableERC20, ERC20 {
```

Comment:
Shouldn't we add is `ILegacyMintableERC20` here too?

Reply:
Do new tokens need to support the legacy interface?

Reply:
Yes, I believe so: https://github.com/ethereum-optimism/optimism/pull/3439/files/910e6bb14be3a59233c5e1cc198d3e59e8fd3bcf#diff-b5041e011abda1d1d726b7fd58224f6cdded883611096c7ad26bd35149020593R540

Reply:
Done in this commit: [`f51a359` (#3439)](https://github.com/ethereum-optimism/optimism/pull/3439/commits/f51a3595efd5ea704a2463e4182a03fda35ba326)